openelevationservice/server/api/querybuilder.py

```diff
@@ -32,19 +32,95 @@ def _getModel(dataset):
     return model
 
 
-def format_PixelAsPoints(result_pixels):
+def format_PixelAsGeoms(result_pixels):
     # format: [ ('(0101000020E61000000000000000C05240D169039D36003D40,202,1,2)',) , ...
-    points = []
+    geoms = []
     heights = []
     for pixel in result_pixels:
         subcolumns = pixel[0].split(",")
-        points.append(subcolumns[0][1: ])
+        geoms.append(subcolumns[0][1: ])
         heights.append(int(subcolumns[1]))
     
-    return func.unnest(literal_column("ARRAY{}".format(points))), \
+    return func.unnest(literal_column("ARRAY{}".format(geoms))), \
            func.unnest(literal_column("ARRAY{}".format(heights)))
 
 
+def polygon_coloring_elevation(geometry, dataset):
+    """
+    Performs PostGIS query to enrich a polygon geometry.
+    
+    :param geometry: Input 2D polygon to be enriched with elevation
+    :type geometry: Shapely geometry
+    
+    :param dataset: Elevation dataset to use for querying
+    :type dataset: string
+    
+    :raises InvalidUsage: internal HTTP 500 error with more detailed description. 
+        
+    :returns: 3D polygon as GeoJSON or WKT
+    :rtype: string
+    """
+    Model = _getModel(dataset)
+
+    # May be a good idea to make this a parameter from request
+    num_ranges = 23
+    range_div = 43
```

Comment:
Explain at the next meeting why this might be a good idea, and maybe we can consider it.